### PR TITLE
[PR] - Universities can signup client

### PR DESCRIPTION
**Pull request by erikwesterberg**
User story- Research Groups Can Post Articles (Client) 

- Set acceptance tests for sign up process.
-Create signup page component and login page component with component test.
-Add Auth Module for Uni login (after sign up)
-University sign-up form must have a required Name field

PT - https://www.pivotaltracker.com/story/show/167890481

### CODE REVIEW

File: cypress/integration/researchGroupCanCreateArticle.spec.js [hunk 1]
```diff
@@ -24,12 +24,12 @@ describe("Research Group can post article", () => {
...
   });
 
-  it("unsuccessfully", () => {
+  it("Article posts unsuccessfully", () => {
     cy.route({
       method: "POST",
       url: "http://localhost:3000/api/v0/articles",
       response: "fixture:unsuccessful_saving_article_response.json",
-      status: 404
+      // status: 404
```
Comment (oliverochman) [inline cypress/integration/researchGroupCanCreateArticle.spec.js]: why is this commented out?

File: src/components/SignupForm.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                }
+              />
+            </Form.Field>
+
+            {this.state.accountType === "University" ? <PaymentForm /> : ""}
+
+            <Button id="submit-account-button" type="submit">
+              Sign Me Up!
+            </Button>
+            {/* After extracting this component to its own page, remove below. . .we don't need to hide the form */}
```
Comment (oliverochman) [inline src/components/SignupForm.js]: please remove comments

Comment (sethnejame) [reply src/components/SignupForm.js]: Done!

File: cypress/integration/researchGroupCanCreateArticle.spec.js (hunk 1 above)
Comment (sethnejame) [reply cypress/integration/researchGroupCanCreateArticle.spec.js]: Silly me.  Fixed!

**Pull request merged**